src/updater.h

```diff
@@ -55,12 +55,21 @@ class MetadataUpdater {
     MetadataStore::Metadata metadata_;
   };
 
+  // A representation of all validation errors.
+  class ValidationError {
```

Comment:
ConfigValidationError to be more specific?

Reply:
Done.